indico/modules/events/persons/__init__.py

```diff
@@ -32,3 +32,16 @@
 def _sidemenu_items(sender, event, **kwargs):
     if event.can_manage(session.user):
         return SideMenuItem('persons', _('Roles'), url_for('persons.person_list', event), section='reports')
+
+
+@signals.get_placeholders.connect_via('roles-email')
```

Comment:
"roles" is just the "displayed" name of this feature, internally it's "event persons", so i'd call this `event-persons-email`